web/src/features/fba/components/ZoneSummaryPanel.tsx

```diff
@@ -10,24 +12,28 @@ const useStyles = makeStyles({
 })
 
 interface Props {
-  className?: string
   selectedZoneID: number | null
+  fireZoneAreas: FireZoneArea[]
 }
 
 const ZoneSummaryPanel = (props: Props) => {
   const classes = useStyles()
-
   if (isNull(props.selectedZoneID)) {
-    return <div></div>
-  } else {
-    return (
-      <div className={props.className}>
-        <div className={classes.wrapper}>
-          <TextField value={props.selectedZoneID} />
-        </div>
-      </div>
-    )
+    return <></>
   }
+
+  return (
+    <Grid container alignItems={'center'} direction={'column'} spacing={2} className={classes.wrapper}>
+      <Grid item>
+        <Typography>Zone ID: {props.selectedZoneID}</Typography>
```

Comment:
shouldn't need this line - the zone name will be displayed

Reply:
Yeah I think the labels will change a lot, I'm gonna rebase off your PR once you get it in and adopt the labels that are there.